### PR TITLE
[res] Fix shape of Net_ZeroDim_001

### PR DESCRIPTION
**Pull request by jinevening**
This adds dynamic shape info to Net_ZeroDim_001.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Related to: https://github.com/Samsung/ONE/issues/13697
Draft PR: https://github.com/Samsung/ONE/pull/13750

### CODE REVIEW

File: res/TensorFlowLiteRecipes/Net_ZeroDim_001/test.recipe [hunk 1]
```diff
@@ -61,6 +72,16 @@ operand {
...
+    dim: 0
+    dim: 8
+    dim: 0
+    dim: 64
+  }
+  shape_signature {
+    dim: -1
+    dim: 8
+    dim: -1
+    dim: 64
```
Comment (jinevening) [inline res/TensorFlowLiteRecipes/Net_ZeroDim_001/test.recipe:84]: This is the shape of output. The output shape should match with the new shape inference logic in #13750.

File: res/TensorFlowLiteRecipes/Net_ZeroDim_001/test.recipe [hunk 2]
```diff
@@ -31,6 +31,11 @@ operand {
     dim: 0
     dim: 512
   }
+  shape_signature {
+    dim: 1
+    dim: -1
+    dim: 512
+  }
```
Comment (seanshpark) [inline res/TensorFlowLiteRecipes/Net_ZeroDim_001/test.recipe:38]: We may have to revise comment

> This was generated by tflchef-reverse from res/TensorFlowTests/NET_0043

Comment (jinevening) [reply res/TensorFlowLiteRecipes/Net_ZeroDim_001/test.recipe:39]: Thanks for pointing out. I've updated the comment.

**Review by seanshpark — APPROVED** [review]
LGTM thank you!

**Pull request merged**